--- parasail/tests/test_verify_traces.c.orig	2025-01-08 16:12:33
+++ parasail/tests/test_verify_traces.c	2025-01-08 16:47:27
@@ -152,7 +152,7 @@
                     unsigned long a = 0;
                     unsigned long b = 1;
                     int8_t *ref_trace_table = NULL;
-                    int *trace_table = NULL;
+                    int8_t *trace_table = NULL;
                     size_t size_a = 0;
                     size_t size_b = 0;
                     k_combination2(pair_index, &a, &b);
@@ -177,8 +177,8 @@
                         saturated += 1;
                         continue;
                     }
-                    ref_trace_table = parasail_result_get_trace_table(reference_result);
-                    trace_table = parasail_result_get_trace_table(result);
+                    ref_trace_table = (int8_t *)parasail_result_get_trace_table(reference_result);
+                    trace_table = (int8_t *)parasail_result_get_trace_table(result);
                     if (reference_result->score != result->score) {
 #pragma omp critical(printer)
                         {
