--- parasail/tests/test_verify_traces.c.orig	2025-01-08 16:12:33
+++ parasail/tests/test_verify_traces.c	2025-01-08 16:12:52
@@ -151,7 +151,7 @@
                     parasail_result_t *result = NULL;
                     unsigned long a = 0;
                     unsigned long b = 1;
-                    int8_t *ref_trace_table = NULL;
+                    int *ref_trace_table = NULL;
                     int *trace_table = NULL;
                     size_t size_a = 0;
                     size_t size_b = 0;
